.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!--
+
+*Description of changes:*
+
+Describe your changes here, linking to an issue if available.
+
+*Licensing*
+
+To maintain the integrity of the project, we require contributions to be licensed under the Apache License Version 2.0. Please acknowledge this by checking the following check-box. By checking the box, you also acknowledge that you have the authority to license your contribution in this manner. You should also ensure that any new file you add contains "# SPDX-License-Identifier: Apache-2.0" in the header.
```

Comment:
Please change:
* Please acknowledge this by checking the following check-box.*
** Please acknowledge this by checking the box below .**

Reply:
done!